### PR TITLE
Event Handler Test Cases added

### PR DESCRIPTION
**Pull request by Aishwarya-Das-ML**

### CODE REVIEW

File: event-handler/source.go
Comment (Dannana-Venkataramana-ML) [inline event-handler/source.go:1]: Please put the source.go files in a new folder

**Comment by Dannana-Venkataramana-ML** [comment]
Since the folder structure is changed the dockerfile need to update, please .dockerignore file to exclude unnecessary files from coping to the docker image and it will reduce multiple COPY commands in Dockerfile

File: event-handler/Dockerfile [hunk 1]
```diff
@@ -15,15 +15,17 @@ ENV RABBITMQ_PASSWORD=$RABBITMQ_PASSWORD
...
-COPY source.go ./
-COPY config.go ./
-COPY go.mod ./
-COPY go.sum ./
+# COPY main.go ./
+# COPY producer.go ./
+# COPY source.go ./
+# COPY config.go ./
+# COPY go.mod ./
+# COPY go.sum ./
```
Comment (Dannana-Venkataramana-ML) [inline event-handler/Dockerfile:23]: Please remove unwanted code instead of commenting

File: event-handler/source.go
Comment (sikhakolanu-venkatesh-ML) [reply event-handler/source.go:1]: moved the source.go into new folder

File: event-handler/Dockerfile (hunk 1 above)
Comment (sikhakolanu-venkatesh-ML) [reply event-handler/Dockerfile]: removed unwanted lines

**Comment by Aishwarya-Das-ML** [comment]
MicrosoftTeams-image

**Review by Dannana-Venkataramana-ML — APPROVED** [review]
LGTM

**Pull request merged**